### PR TITLE
Support for CentOS

### PR DESCRIPTION
**Pull request by mrcdb**
This PR adds support for both bare-metal and nested Docker deployment of the emulator to a CentOS host.

### CODE REVIEW

**Review by mpeuster — CHANGES_REQUESTED** [review]
PR looks fine for me.
The only thing you should do, is adding copyright headers to all the new files. This is a must have to get code into the OSM repos (they have automatic license checks). It needs to be Apache 2.0 ... You can check existing files as example.

Don't get confused by the "Codacy" tool. I forgot to deactivate it. It should not run :-)

**Comment by mrcdb** [comment]
I will do it now! Thanks for the check :)

**Review by mpeuster — APPROVED** [review]
Great! Now everything looks fine.

File: src/emuvim/dcemulator/net.py [hunk 1]
```diff
@@ -864,7 +864,12 @@ def _set_flow_entry_dpctl(self, node, switch_inport_nr, switch_outport_nr, **kwa
     # start Ryu Openflow controller as Remote Controller for the DCNetwork
     def startRyu(self, learning_switch=True):
         # start Ryu controller with rest-API
-        python_install_path = site.getsitepackages()[0]
+        site_packages = site.getsitepackages()
+
+	python_install_path = ''
+	for site_package in site_packages:
```
Comment (mpeuster) [inline src/emuvim/dcemulator/net.py]: one other small issue here: it should be spaces not tabs to indent the blocks

**Review by mpeuster — CHANGES_REQUESTED** [review]

**Review by mpeuster — APPROVED** [review]

**Pull request closed without merging**